blog/src/content/blog/the-substrate-is-installable.md

```
@@ -0,0 +1,172 @@
+iii worker add iii-cron           # cron scheduling
+iii worker add iii-http           # HTTP endpoints
+iii worker add iii-observability  # OpenTelemetry traces, metrics, logs
+iii worker add iii-sandbox        # hardware-isolated microVMs
+iii worker add iii-bridge         # connect to another iii instance
+```
+
+That's nine commands. That is also, as far as I can tell, every box in the "people keep independently rebuilding the same surrounding layer" list, expressed as the same primitive.
+
+It keeps going. The agent layer itself, on the registry today:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)